Lesson_Materials/Lecture_4_Handling_Errors/index.html

```diff
@@ -355,7 +385,10 @@
 					</div>
 					<div class="bottom-bullets" data-markdown>
 						* Any SYCL application can be debugged on the host device by switching the queue for a host queue
-						* By replacing the device selector for the host_selector will ensure that the queue submits all work to the host device
+						* Replacing the device selector for the host_selector
+							will ensure that the queue submits all work to the host device
+						* SYCL 2020 provides an `aspect_selector`,
```

Comment:
Maybe we should add an example of using the `aspect_selector` for finding a `host_debuggable` aspect to replace the `host_selector`?

Reply:
Done in https://github.com/codeplaysoftware/syclacademy/pull/42/commits/4bf10368e0e8685d7d0f66b7e6f9c128eca24bea. The wording might be a bit awkward because of the lesser guarantees, but I think it still works, let me know.